### PR TITLE
Automation of pipelines-plugin test (create-from-add-options feature file)

### PR DESCRIPTION
**Pull request by SparshKesari**
**Description:**

- Automation of create-from-add-options.feature file to-do tests
    - Create a workload with pipeline from Docker file (P-01-TC06) - (frontend changes - deploy locally)

**Jira Id:** [ODC-6577](https://issues.redhat.com/browse/ODC-6577)

**Pre-conditions/setup:**

**Checks for approving Epic scenarios Automation PR:**

- [x] Execute the @to-do tagged gherkin scripts manually
- [x] Convert the @to-do gherkin scripts to cypress automation scripts
- [x] Once scripts are automated, replace tag @to-do with @epic-number
- [ ] Execute the scripts in Remote cluster

**Refactoring criteria for approving Automation PR:**

- [ ] update the test cases count in [Automation status confluence Report](https://docs.jboss.org/display/ODC/Automation+Status+Report)

**Execution Commands:**
Example: 
```
oc login -u kubeadmin -p $(cat /path/to/install-dir/auth/kubeadmin-password)
source ./contrib/oc-environment.sh
./bin/bridge

cd frontend
oc login ...
yarn run test-cypress-pipelines

```

**Screen shots**:

image

**Browser conformance**:

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by sanketpathak** [comment]
Tests are working in my local
Screenshot from 2022-03-23 15-38-51

File: frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-add-options.feature [hunk 1]
```diff
@@ -3,10 +3,10 @@ Feature: Create Pipeline from Add Options
               As a user, I want to create, edit, delete and view the pipeline
 
         Background:
-            Given user has created or selected namespace "aut-pipelines-add-options"
+            Given user has created or selected namespace "aut-pipelines"
```
Comment (sanketpathak) [inline frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-add-options.feature:6]: Why have you changed the name of the namespace? Is there an issue due to it

Comment (SparshKesari) [reply frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-add-options.feature]: Earlier I was facing some issues with the namespace. Will change the namespace back to the previous one

**Comment by SparshKesari** [comment]
Closing Favour of https://github.com/openshift/console/pull/11291

**Pull request closed without merging**